### ISSUE TITLE
Add action buttons for each file

### ISSUE THREAD

**Issue by aishmitk** [NCI-GDC/dtt-ui#7]
User may want to download only one file which can be done if each file has its own download button

**Comment by Fgerthoffert** [comment]
I would rephrase this ticket to: "Add action buttons for each files", action buttons being:
- [ ] Pause/Resume: Stop execution of a running download or skip download if not running yet. Clicking again would revert to previous state. Icon to change based on state
- [ ] Delete: Remove download from queue (garbage icon)
- [ ] Up: Move download up the queue and start downloading (arrow icon)

**Comment by aishmitk** [comment]
37c5fed

**Comment by aishmitk** [comment]
37c5fed
